packages/sources/web-components/src/components/vtmn-text-input/vtmn-text-input.tsx

```diff
@@ -0,0 +1,119 @@
+import { Component, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'vtmn-text-input',
+  styleUrl: 'vtmn-text-input.pcss',
+  shadow: false,
+})
+export class VtmnTextInput {
+  /**
+   * The id of the text input.
+   * @default 'my-id'
+   */
+  @Prop() id: string = 'my-id';
```

Comment:
No default value for the "id" please. It's dangerous because we will probably have several apps with several same ids in HTML pages. For me `id` is undefined by default.